src/util/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+add_library(bitcoin_util STATIC EXCLUDE_FROM_ALL
+  asmap.cpp
+  bip32.cpp
+  bytevectorhash.cpp
+  check.cpp
+  error.cpp
+  fees.cpp
+  getuniquepath.cpp
+  hasher.cpp
+  message.cpp
+  moneystr.cpp
+  rbf.cpp
+  readwritefile.cpp
+  settings.cpp
+  serfloat.cpp
+  sock.cpp
+  spanparsing.cpp
+  strencodings.cpp
+  string.cpp
+  syscall_sandbox.cpp
+  syserror.cpp
+  system.cpp
+  thread.cpp
+  threadinterrupt.cpp
+  threadnames.cpp
+  time.cpp
+  tokenpipe.cpp
+  ../chainparamsbase.cpp
```

Comment:
Heh, awkward. Motivation to move things to their proper homes? :)

Reply:
> Heh, awkward. Motivation to move things to their proper homes? :)

Exactly :)